graphql/README.md

```diff
@@ -1,25 +1,12 @@
 # GraphQL
 
-A guide for GraphQL, a place to document questions, answers, best practices,
-smells, solutions to smells, etc. as we discover how to do GraphQL well.
+A guide for building GraphQL servers and clients.
 
-## Resources
+If you're just getting started, check out the [official GraphQL tutorial].
 
-### Learning
+[official GraphQL tutorial]: https://graphql.org/learn/
 
-A curated list of resources for learning GraphQL.
```

Comment:
I tightened this up a bit as our guides are generally succinct and serve as reference points rather than being tutorials. Some of these original resources also included non-public links, which makes them less valuable as a community reference.

Reply:
We often have people join the `#graphql` channel and ask where to start learning it, how did we learn it, etc.? Having those links somewhere would be useful.

Reply:
Okay, that's reasonable. I restored this section in 3c89384.